addons/sale_timesheet/__manifest__.py

```diff
@@ -50,8 +50,12 @@
             'sale_timesheet/static/tests/tours/*',
             'web/static/lib/hoot-dom/**/*',
         ],
-        'web.qunit_suite_tests': [
+        'web.assets_unit_tests': [
             'sale_timesheet/static/tests/**/*',
+            ('remove', 'sale_timesheet/static/tests/tours/**/*'),
+        ],
+        'web.qunit_suite_tests': [
+            'sale_timesheet/static/tests/tours/**/*',
```

Comment:
Unless this module is doing something very confusing with its file or filenames, tours shouldn't be included in qunit bundle.